src/app/project-notifications/add-edit-project-notification/add-edit-project-notification.component.ts

```diff
@@ -170,7 +178,20 @@ export class AddEditProjectNotificationComponent implements OnInit, OnDestroy {
   }
 
   private buildForm(data) {
-    const natureDisabled = data.trigger !== 'Greenhouse Gases';
+    // data will be csv in trigger attribute, so split into names
+    // then add the constant values into 'triggers' that match
+    // also, trigger the natureDisabled flag as needed
+
+    this.triggers = [];
+    let dataTriggers =  data.trigger.split(',');
+
+    Constants.NOTIFICATION_TRIGGERS.forEach(trigger => {
+      if (dataTriggers.includes(trigger.name)) {
+        this.triggers.push(trigger);
+      }
+    });
+
+    const natureDisabled = !data.trigger.split(',').includes('Greenhouse Gases (modification)') && !data.trigger.split(',').includes('Greenhouse Gases (new project)');
```

Comment:
Could we put these strings in the config file with all the other strings for the nature field?

Reply:
These aren't nature values, they're trigger values. I can pull them from the constants as well, but I'd still need to search for the specific name at some point so it'll be hard coded here, or hard-coded elsewhere

Reply:
yeah fair enough!